src/index.tsx

```diff
@@ -0,0 +1 @@
+export { default } from "./platformIcon";
```

Comment:
maybe the` platformIcon.tsx ` should be the index?

Reply:
I kind of like having index.tsx in npm packages as an entry file that reexports all the components in the package to dictate the public API.  I agree that with one component it's not that necessary, but I would like to keep it this way so that it's easier to add other things in there in the future if needed.